README.md

```diff
@@ -13,9 +13,16 @@ This directory contains the following helm charts.
 
 ## Prerequisites
 
-* Kubernetes 1.19+
 * Helm 3.5+
 
+## Supported Kubernetes version
+
+* Kubernetes 1.25
```

Comment:
Better to be simplified like this?
```
* 1.[21-25].x
```

Reply:
Thank you for your suggestion!

It is simple and we can maintain it easily. So, I think it is better from perspective of developer/maintainer (i.e., us).
However, I think there is a possibility that the regular expressions is a bit confusing.

The `list` is easy to understand with a quick glance.
So, from the perspective of users, I think it is better to describe supported version as a list.

What do you think?